backend/src/data/migrations/20220206164137_edit_masters_table.ts

```diff
@@ -0,0 +1,16 @@
+import { Knex } from 'knex';
+import { TableName } from '~/common/enums/enums';
+
+const TABLE_NAME = TableName.MASTERS;
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table(TABLE_NAME, function (table) {
+    table.uuid('tenant_id').references('id').inTable(TableName.TENANTS);
```

Comment:
should be notNullable

Reply:
ok, but the scheme indicated --> https://prnt.sc/26pzdde

Reply:
got it. in this case, we don't need it 👍